pkgs/applications/science/electronics/gerbv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchgit, pkgconfig, gettext, libtool, automake, autoconf, cairo, gtk, desktop_file_utils}:
+
+stdenv.mkDerivation rec {
+  name = "gerbv-${version}";
+  version = "2015-10-08";
+
+  src = fetchgit {
+    url = git://git.geda-project.org/gerbv.git;
+    rev = "76b8b67bfa10823ce98f1c4c3b49a2afcadf7659";
+    sha256 = "1l2x8sb1c3gq00i71fdndkqwa7148mrranayafqw9pq63869l92w";
+  };
+
+  buildInputs = [ pkgconfig gettext libtool automake autoconf cairo gtk desktop_file_utils ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+#    postConfigure = ''
+# #    make
+#     sleep 5
+#     echo hey mog
+#     cd po
+#     msginit -l ru
+#     cd ..
+#     touch man/gerbv.ru.1.in.po
+#   '';
+
+  meta = with stdenv.lib; {
+    description = "Gerbv -- A Gerber (RS-274X) viewer";
+    homepage = http://gerbv.geda-project.org/;
+    maintainers = with maintainers; [ mog ];
+    platforms = platforms.linux;
+    license = licenses.gpl2;
+    priority = 10;
```

Comment:
nix-env -f $NIXPKGS  -i gerbv
replacing old ‘gerbv-2015-10-08’
installing ‘gerbv-2015-10-08’
building path(s) ‘/nix/store/1cyz0acybk9440ylw69s72xiq75wyh12-user-environment’
Wide character in die at /nix/store/jjgyvh88yw37gg5ad5lj1z0qdv31d7jy-buildenv.pl line 79.
collision between ‘/nix/store/q8d3c4fhjr1sxhbljdcxfm7s19zjlvzj-hexchat-2.10.2/share/applications/mimeinfo.cache’ and ‘/nix/store/5p57j611z8nj01axvk68qbcqjj0xaja0-gerbv-2015-10-08/share/applications/mimeinfo.cache’; use ‘nix-env --set-flag priority NUMBER PKGNAME’ to change the priority of one of the conflicting packages
builder for ‘/nix/store/7izpjwdqapidp8ik9ff231bs31xrmk37-user-environment.drv’ failed with exit code 2
error: build of ‘/nix/store/7izpjwdqapidp8ik9ff231bs31xrmk37-user-environment.drv’ failed